src/components/Registration/RegPeopleAndRoles.vue

```diff
@@ -158,7 +160,19 @@ export default class RegPeopleAndRoles extends Mixins(PeopleRolesMixin) {
   // NB: see mixin for common methods, getters, etc.
   //
 
-  private addOrgPerson (roleType: RoleTypes, partyType: PartyTypes): void {
+  protected async addOrgPerson (roleType: RoleTypes, partyType: PartyTypes): Promise<void> {
```

Comment:
`protected` means the template (and any object that inherits this class) can see this method, but it can't be seen externally.

Ref: https://www.typescriptlang.org/docs/handbook/2/classes.html#member-visibility

Reply:
Very cool
For my own knowledge what is the purpose?
ie why Protected vs Private or Public?

Reply:
This is a class, i.e., an object-oriented structure. It can contain some things that external code should not be able to see or change.

In this case, I don't want `addOrgPerson` to be publicly accessible, but if I make it private then my IDE's plugins tell me it's not used. It's actually used in the template only. Protected meets both these conditions.